app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -45,10 +45,8 @@ def authenticate_user
 
   def handle_second_factor_locked_user(type)
     analytics.track_event(Analytics::MULTI_FACTOR_AUTH_MAX_ATTEMPTS)
-
-    sign_out
-
     render 'two_factor_authentication/shared/max_login_attempts_reached', locals: { type: type }
+    sign_out
```

Comment:
@jessieay I was able to make [this test](https://github.com/18F/identity-idp/blob/master/spec/features/two_factor_authentication/sign_in_spec.rb#L114-L143) fail by changing the `session_check_delay` to `'1'`, and adding these 2 lines below line 128:
```ruby
sleep 1
 expect(page).to have_content(/4:5\d/)
```
Is there a better way to test this?